data/spells/scripts/support/animate_dead_rune.lua

```diff
@@ -18,6 +18,6 @@ function onCastSpell(creature, variant, isHotkey)
 	end
 
 	creature:getPosition():sendMagicEffect(CONST_ME_POFF)
-	creature:sendCancelMessage(RETURNVALUE_NOTPOSSIBLE)
+	creature:sendTextMessage(MESSAGE_STATUS_SMALL, Game.getReturnMessage(RETURNVALUE_NOTPOSSIBLE))
```

Comment:
It's not from compat, `sendCancelMessage` is different because it gets the message from a predefined list.